CONTRIBUTORS.md

```diff
@@ -2,6 +2,8 @@ Cerner Corporation
 
 - Matt Butler [@matt-butler]
 - Tao Zhang [@windse7en]
```

Comment:
If we're completely re-writing this, are they technically still contributors?

Reply:
🤷‍♀️

Reply:
On this page they would be https://github.com/cerner/terra-ui.com/graphs/contributors

Reply:
Not a big deal, more of a philosophical question than anything.

Reply:
Let's leave them as contributors